docs/source/quantization_weight_only.md

```diff
@@ -76,9 +85,8 @@ conf = PostTrainingQuantConfig(
             },
         },
     },
-    ### GPTQ is WIP
     recipes={
-        # 'gptq_args':{'percdamp': 0.01},
+        # 'gptq_args':{'percdamp': 0.01, 'actorder':True, 'block_size': 128, 'nsamples': 128, 'use_full_length': False},
```

Comment:
better to change "use full length " to "use max length"?

Reply:
resolved